pkg/document/json/root_test.go

```diff
@@ -26,33 +26,53 @@ import (
 	"github.com/yorkie-team/yorkie/testhelper"
 )
 
-func registerTextElement(fromPos, toPos *json.RGATreeSplitNodePos, root *json.Root, text json.TextElement) {
+func registerRemovedNodeTextElement(fromPos, toPos *json.RGATreeSplitNodePos, root *json.Root, text json.TextElement) {
 	if fromPos.Compare(toPos) != 0 {
 		root.RegisterRemovedNodeTextElement(text)
 	}
 }
 
 func TestRoot(t *testing.T) {
-	t.Run("garbage collect for text test", func(t *testing.T) {
+	t.Run("garbage collection for array test", func(t *testing.T) {
```

Comment:
This test seems to be out of scope for this PR. Any reason added in this PR?

Reply:
Disconnecting `node.prev` and `node.next` is missing when purging a node from RGATreeList. As I added related tests in this PR, I changed the names of other adjacent tests as well.